src/styles/index.css

```
@@ -0,0 +1,408 @@
+  letter-spacing: 1.25px;
+  text-align: left;
+  color: rgba(255, 255, 255, 1);
+}
+
+.purchase__btn-text {
+  box-sizing: border-box;
+  font-family: Roboto;
+  width: 42px;
+  height: 24px;
+  padding: 4px 6px 4px 6px;
+  margin: 0;
+}
+
+.purchase-lotto__text {
+  position: absolute;
+  width: 310px;
+  height: 24px;
+  top: 196px;
+  left: 16px;
+  margin: 0;
+}
+
+.purchase-lotto__list {
+  position: relative;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 238px;
+  height: 276px;
+  top: 228px;
+  left: 16px;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  gap: 4px 0;
+  overflow: auto;
+  overflow-x: hidden;
+}
+
+.purchase-lotto__list li {
+  display: flex;
+  align-items: center;
+  height: 36px;
+  margin: 0;
+}
+
+.purchase-lotto__admit-one {
+  width: 34px;
+  height: 36px;
+  font-size: 34px;
+  font-weight: 600;
+  line-height: 36px;
+  letter-spacing: 0px;
+  text-align: left;
+  color: rgba(0, 0, 0, 1);
+}
+
+.purchase-lotto__list li p {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0 0 0 8px;
+  width: 196px;
+  height: 24px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.5px;
+  text-align: left;
+  color: rgba(0, 0, 0, 1);
+}
+
+.winning-lotto {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 382px;
+  height: 104px;
+  top: 528px;
+  left: 16px;
+}
+
+.winning-lotto__text {
+  font-size: 15px;
+  line-height: 24px;
+  letter-spacing: 0.5px;
+  margin: 0;
+}
+
+.winning-lotto-input-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.winning-lotto-numbers {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 244px;
+  height: 68px;
+}
+
+.winning-lotto-numbers__input-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.winning-lotto-bonus-number {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: end;
+  width: 76px;
+  height: 68px;
+}
+
+.winning-lotto-numbers__input,
+.winning-lotto-bonus-number__input {
+  box-sizing: border-box;
+  width: 34px;
+  height: 36px;
+  border-radius: 4px;
+  border: 1px solid rgba(180, 180, 180, 1);
+}
+
+.winning-lotto__btn {
+  position: absolute;
+  cursor: pointer;
+  width: 382px;
+  height: 36px;
+  top: 659px;
+  left: 16px;
+  padding: 6px 6px 6px 8px;
+  border-radius: 4px;
+  background: rgba(78, 91, 166, 1);
+}
+
+.winning-lotto__btn-text {
+  width: 368px;
+  height: 24px;
+  padding: 4px 6px 4px 6px;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 1.25px;
+  color: rgba(255, 255, 255, 1);
+  margin: 0;
+}
+
+.winning-result-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+}
+
+.winning-result-modal_hidden {
+  display: none;
+}
+
+.winning-result-modal_visible {
+  display: block;
+}
```

👍 👍 
이거는 근데 `winning-result-modal`안에서만 사용 될 필요 없을 것 같아요! 구조 상관없이 재활용 가능한 클래스로 보여서 클래스명을 아래와 같이 변경해도 괜찮아 보입니다 :)
```suggestion
.hidden {
  display: none;
}

.visible {
  display: block;
}
```

---

bem을 지키려는게 우선되어야하나? 라는 생각에 우선 다 적용해보았는데 반복되는 부분은 이렇게 모아도 되는군요! 반복을 줄이는 것이 먼저인가 아니면 다 속성을 표현해야하는가에 대한 고민이 있었는데 좋은 피드백 감사합니다!